intro.html

```diff
@@ -383,11 +413,99 @@ <h3>Let’s start with an example</h3>
 
 <h2>Types of testing</h2>
 
-<p>TODO: Add paragraphs about acceptance testing, performance testing,
- load testing, regression testing, test driven development, and/or
- behavior-driven development (JBehave, Capybara, & Robot Framework),
- with how they relate to Selenium.
-
+<h3>Acceptance testing</h3>
+<p>This type of test is done in order to determine if a product's
+   feature matches it's requirements. 
+   This generally involve the customer's feedback or specification.</p>
+<p>For web applications, the automation of this testing can be done
+   directly with Selenium by simulating a user expected behavior in them.
+   This simulation could be done by record/playback or through the
+   different supported languages as explained in this documentation.</p>
+<p>Note: Acceptance testing is a subtype of <b>functional testing</b>,
+   which some people might also refer to.
+</p>
+
+<h3>Functional testing</h3>
+<p>This type of test is done in order to determine if a product's
+  feature functions properly, without issues.</p>
+<p>This generally include: the tests work without errors 
+  (404, exceptions...), in an usable way (right redirections),
+    in an accessible way and matching it's specifications
+     (see <b>acceptance testing</b> above).</p>
+<p>For web applications, the automation of this testing can be
+   done directly with Selenium by simulating expected returns.
+   This simulation could be done by record/playback or through
+   the different supported languages as explained in this documentation.</p>
+
+<h3>Performance testing</h3>
```

Comment:
why include performance testing info at all?  I'd leave this entire section out.

Reply:
It was part of the TODO. Also, Selenium takes part on it, so why not?
Also, I'm adding these changes as part of a commit for the new documentation, please follow up there, if that's OK :)